src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -1062,12 +1065,12 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancellat
 
             cancellationToken.ThrowIfCancellationRequested();
 
-            string outputName = GetOutputFileName(compilation, cancellationToken);
+            string outputName = GetOutputFileName(compilation, cancellationToken)!;
```

Comment:
This is the hardest to grok and I want to come back and refactor this later. For now just being honest about the `null` intent here.

Reply:
Is there a bug tracking?

---
In reply to: [502079520](https://github.com/dotnet/roslyn/pull/48460#discussion_r502079520) [](ancestors = 502079520)

Reply:
https://github.com/dotnet/roslyn/issues/48599